discord/public_flags.py

```diff
@@ -0,0 +1,96 @@
+from .enums import UserFlags
+
+
+class PublicFlags:
```

Comment:
```suggestion
class PublicUserFlags:
```

This is actually not how flags are done in the library.

Please see `flags.py` for an example on how flags are done.